src/EnergyPlus/SetPointManager.cc

```diff
@@ -7710,7 +7710,7 @@ void DefineCondEntSetPointManager::calculate(EnergyPlusData &state)
     // na
 
     // Using/Aliasing
-    using Curve::CurveValue;
+    using EnergyPlus::Curve::CurveValue;
```

Comment:
Really?

Reply:
The namespace changes were just a quick fix on my Windows machine to get the build to succeed.  There were build failures due to ambiguous types that were not showing on Linux.  So that was the quickest way to get the build passing.  I'll go back through and just eliminate the `using` entirely for these cases.